automation/identifiers/buttons/ActionButtons.ts

```diff
@@ -0,0 +1,4 @@
+export default class ActionButtons {
```

Comment:
Please, make the buttons in 1 'Buttons' identifier class, and in it split it by regions, because dividing it for 'Actions' or 'MainTable' has no sense and no traceability